draft-kazuho-httpbis-priority.md

```diff
@@ -449,6 +449,20 @@ frame header MUST be zero (0x0).
 ~~~
 {: #fig-h2-reprioritization-frame title="HTTP/2 PRIORITY_UPDATE Frame Payload"}
 
+The PRIORITY_UPDATE frame payload has the following fields:
+
+R: 
+: A reserved 1-bit field. The semantics of this bit are undefined,
+and the bit MUST remain unset (0x0) when sending and MUST be
```

Comment:
I think ideally this would all be indented, and below

Reply:
Done in 5bb07f7.